heaps/heap_sort.py

```diff
@@ -1,8 +1,19 @@
-
+from heaps.min_heap import MinHeap
 
 def heap_sort(list):
     """ This method uses a heap to sort an array.
-        Time Complexity:  ?
-        Space Complexity: ?
+        Time Complexity:  O(n)
+        Space Complexity: O(n)
     """
```

Comment:
👍  However the time complexity is O(n log n) because you are adding n elements to the heap and each addition takes O(log n) time.